### DISCUSSION TITLE
2011/04/05/%E6%88%91%E7%9A%84%E6%97%A7%E7%85%A7%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#323] [Announcements]
# 2011/04/05/%E6%88%91%E7%9A%84%E6%97%A7%E7%85%A7%E7%89%87

今天扫了几张我的旧照片，跟豆豆比较一下:)

https://blog.qizhen.xyz/2011/04/05/%E6%88%91%E7%9A%84%E6%97%A7%E7%85%A7%E7%89%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

lizzie
2011年04月6日 17:10 编辑
你小时候眼睛好像比现在大点嘿嘿

回复
勺子
2011年04月6日 12:59 编辑
哈哈，那个小花裤最搞了。。。

回复
Haiying
2011年04月5日 16:34 编辑
你小时候那么帅啊!!啊阮妈妈真漂亮

回复
